<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QDialog" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dashboard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="nav_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>51</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(127, 130, 181); 
border: none;</string>
   </property>
   <widget class="QPushButton" name="salary_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
 background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="performance_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
 background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="attendance_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
 background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="logout_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
 background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="emp_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../icons/employee.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="home_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>1311</width>
     <height>761</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <widget class="QWidget" name="home_page">
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>491</width>
       <height>261</height>
      </rect>
     </property>
     <widget class="QLabel" name="empInfo_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;
font-size: 17px;
font-weight: 1000;
</string>
      </property>
      <property name="text">
       <string>Employee Information</string>
      </property>
     </widget>
     <widget class="QLabel" name="empID_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;

</string>
      </property>
      <property name="text">
       <string>Employee ID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="empName_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;

</string>
      </property>
      <property name="text">
       <string>Employee name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="empEmail_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;

</string>
      </property>
      <property name="text">
       <string>Employee email:</string>
      </property>
     </widget>
     <widget class="QLabel" name="empDept_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;
</string>
      </property>
      <property name="text">
       <string>Employee department:</string>
      </property>
     </widget>
     <widget class="QLabel" name="empMobile_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;

</string>
      </property>
      <property name="text">
       <string>Employee mobile: </string>
      </property>
     </widget>
     <widget class="QLabel" name="empCnic_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>431</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;

</string>
      </property>
      <property name="text">
       <string>Employee CNIC:</string>
      </property>
     </widget>
     <widget class="QLabel" name="empExp_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;

</string>
      </property>
      <property name="text">
       <string>Employee experience:</string>
      </property>
     </widget>
     <widget class="QLabel" name="empAge_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;

</string>
      </property>
      <property name="text">
       <string>Employee age:</string>
      </property>
     </widget>
     <widget class="QLabel" name="empGender_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;

</string>
      </property>
      <property name="text">
       <string>Employee gender:</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="profile_img_1">
     <property name="geometry">
      <rect>
       <x>1150</x>
       <y>0</y>
       <width>160</width>
       <height>120</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="welcome_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font-size: 20px;
font-weight: 1000;</string>
     </property>
     <property name="text">
      <string>Welcome Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="quote_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>630</y>
       <width>891</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Downloads/output-onlinepngtools (2).png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="task_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>501</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font-size: 17px;
font-weight: 1000;
</string>
     </property>
     <property name="text">
      <string>Current task: </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="salary_page">
    <widget class="QLabel" name="profile_img_2">
     <property name="geometry">
      <rect>
       <x>1150</x>
       <y>0</y>
       <width>160</width>
       <height>120</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font-size: 20px;
font-weight: 1000;
</string>
     </property>
     <property name="text">
      <string>Salary</string>
     </property>
    </widget>
    <widget class="QLabel" name="tax_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font-size: 17px;
font-weight: 1000;
</string>
     </property>
     <property name="text">
      <string>Tax: </string>
     </property>
    </widget>
    <widget class="QLabel" name="bonus_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font-size: 17px;
font-weight: 1000;
</string>
     </property>
     <property name="text">
      <string>Bonus:</string>
     </property>
    </widget>
    <widget class="QLabel" name="salary_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font-size: 20px;
font-weight: 1000;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="salary_graph_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>871</width>
       <height>391</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Pictures/salary_graph.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="attendance_page">
    <widget class="QLabel" name="profile_img_3">
     <property name="geometry">
      <rect>
       <x>1150</x>
       <y>0</y>
       <width>160</width>
       <height>120</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>551</width>
       <height>71</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;
font-size: 15px;</string>
       </property>
       <property name="text">
        <string>Total leaves</string>
       </property>
      </widget>
      <widget class="QLabel" name="total_leaves_label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;
font-size: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>141</width>
        <height>71</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>9</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;
font-size: 15px;</string>
       </property>
       <property name="text">
        <string>Approved leaves</string>
       </property>
      </widget>
      <widget class="QLabel" name="approved_leaves_label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;
font-size: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;
font-size: 15px;</string>
       </property>
       <property name="text">
        <string>Rejected leaves</string>
       </property>
      </widget>
      <widget class="QLabel" name="rejected_leaves_label">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;
font-size: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_6" native="true">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>0</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <widget class="QLabel" name="widget_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;
font-size: 15px;</string>
       </property>
       <property name="text">
        <string>Pending</string>
       </property>
      </widget>
      <widget class="QLabel" name="pending_leaves_label">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;
font-size: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font-size: 20px;
font-weight: 1000;</string>
     </property>
     <property name="text">
      <string>Attendance</string>
     </property>
    </widget>
    <widget class="QPushButton" name="leave_btn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(127, 130, 181); 
border: none;
color: white;
font-weight: 500;</string>
     </property>
     <property name="text">
      <string>Apply for leave</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font-size: 20px;
font-weight: 1000;</string>
     </property>
     <property name="text">
      <string>87.01%</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="performance_page">
    <widget class="QLabel" name="profile_img_4">
     <property name="geometry">
      <rect>
       <x>1150</x>
       <y>0</y>
       <width>160</width>
       <height>120</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font-size: 20px;
font-weight: 1000;
</string>
     </property>
     <property name="text">
      <string>Performance</string>
     </property>
    </widget>
    <widget class="QLabel" name="performance_graph_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>641</width>
       <height>481</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Pictures/performance_graph1.png</pixmap>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
